NurVo_Frontend/src/pages/StudyPage.tsx

```diff
@@ -1,293 +1,73 @@
-  // 카드 앞면, 뒷면
-  const [change, setChange] = useState(new Array(textArray.length).fill(false));
-  // 다음 단계 넘어가기
-  const [isDisabled, setIsDisabled] = useState(false);
-  // 현재 카드 번호
-  const [currentIndex, setCurrentIndex] = useState(0);
-  // const swiperRef = useRef(null);
-  const swiperRef = useRef<Swiper>(null);
-
-  useEffect(() => {
-    if (isDisabled) {
-      setTextArray(nextTextArray);
-      setCount(-1);
-      setCurrentIndex(-1);
-      setNextTextArray([]);
-      setIsDisabled(false);
-      change.fill(false);
-    }
-  }, [isDisabled]);
+  const [action, setAction] = useState('');
+  const [alertOpen, setAlertOpen] = useState(false);
 
-  // alert 창
-  const showAlert = () => {
-    Alert.alert(
-      '한줄학습',
-      '목표하신 문장을 모두 복습하셨습니다.',
-      [
-        { text: '확인', onPress: () => navigation.navigate('SelectText') },
-      ],
-      { cancelable: false }
-    );
+  const handleChangeAction = (value: React.SetStateAction<string>) => {
+    setAction(value);
   };
 
-  // 카드 뒤집기
-  const turnCard = (index: number) => {
-    const updatedChange = [...change];
-    updatedChange[index] = !updatedChange[index];
-    setChange(updatedChange);
+  // 제거
+  const remove = () => {
+    setProgress(prev => prev + 1);
+    setAction('remove');
   };
 
-  // 화면 전환
-  const roof = () => {
-    if (swiperRef.current) {
-      const swiperState = (swiperRef.current as any).state;
-      const totalSlides = swiperState.total;
-      if (currentIndex === totalSlides - 1) {
-        if (swiperRef.current) {
-          swiperRef.current.scrollTo(0, false);
-          setIsDisabled(true);
-        }
-      } else {
-        if (swiperRef.current) {
-          swiperRef.current.scrollBy(1, true);
-        }
-      };
-    }
+  // 유지
+  const keep = () => {
+    setAction('keep');
   };
 
-  // 다음 단계 문장 제거
-  const removeBtn = () => {
-    if (currentIndex < textArray.length) {
-      setCount(prev => prev + 1);
-      setProgress(prev => prev + 1);
-      setCurrentIndex(prev => prev + 1);
-    }
-    roof();
-  };
-
-  // 다음 단계 문장 저장
-  const keepBtn = () => {
-    if (currentIndex >= 0) {
-      const newArray = nextTextArray;
-      newArray.push(textArray[currentIndex]);
-      setNextTextArray(newArray);
-    }
-    if (currentIndex < textArray.length) {
-      setCount(prev => prev + 1);
-      setCurrentIndex(prev => prev + 1);
-    }
-    roof();
-  };
-
-  // TTS 실행
-  const Speaker = (sample: string) => {
-    Tts.speak(sample);
-  };
-
-  // 진행바 스타일
-  const progressStyle: ViewStyle = {
-    width: `${(progress / Data.length) * 100}%`,
-    height: '100%',
-    backgroundColor: 'rgba(98, 196, 150, 1)',
-  };
-
-  // 100% 진행시 Aler창이 뜨고 저장 문장 페이지로 이동
-  if (count == -1 && textArray.length == 0) {
-    showAlert();
-    setTextArray([]);
-    setNextTextArray([]);
-    setCount(-2);
-  };
+  // Library로 이동
+  const handlePage = () => {
+    navigation.navigate('Library');
```

Comment:
```suggestion
   navigation.popToTop();
```
처음 화면으로 돌아가는 기능이라서 네비게이션 보다는 popToTop()기능이 더 적절한 것 같아요!

Reply:
알려주셔서 감사합니다!! 수정했습니다.